css/style.css

```diff
@@ -0,0 +1,3111 @@
+}
+
+.inside {
+  padding-top: 37px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
+  min-width: 1440px;
+  max-width: 1440px;
+}
+
+.inside__title {
+  font-size: 70px;
+  font-family: "MontserratSemiBold";
+  padding: 64px 80px;
+  margin: 0;
+  color: #E51937;
+}
+
+.inside__image {
+  position: absolute;
+  width: 80%;
+  z-index: -1;
+  bottom: 0;
+  height: 90%;
+}
+
+.inside__description {
+  background: #e51937;
+  color: white;
+  width: 26%;
+  padding: 51px 44px 63px 52px;
+}
+
+.description-inside__title {
+  font-family: "MontserratSemiBold";
+  display: flex;
+  flex-direction: row;
+  font-size: 30px;
+  width: 100%;
+  text-align: justify;
+  margin: 0;
+}
+
+.description-inside__text {
+  font-family: "MontserratMedium";
+  font-size: 15px;
+  width: 96%;
+  padding-bottom: 50px;
+  margin-bottom: 0;
+}
+
+.description-inside__display {
+  display: flex;
+  flex-direction: row;
+}
+
+.display-inside__icon {
+  position: relative;
+}
+
+.display-inside__image {
+  position: relative;
+  z-index: 0;
+}
+
+.display-inside__icon-wrapper {
+  position: absolute;
+  z-index: 1;
+  top: 33px;
+  left: 13px;
+}
+
+.icon-wrapper__pulse {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 9px;
+  height: 9px;
+  color: gray;
+  background: white;
+  border-radius: 50%;
+  animation: radial-pulse 1s infinite;
+}
+
+@keyframes radial-pulse {
+  0% {
+    box-shadow: 0 0 0 0 rgba(0, 0, 0, 0.5);
+  }
+
+  100% {
+    box-shadow: 0 0 0 12px rgba(0, 0, 0, 0);
+  }
+}
+
+.display-inside__desc {
+  display: flex;
+  flex-direction: row;
+  padding-left: 19px;
+  font-family: "MontserratSemiBold";
+  padding-bottom: 1px;
+}
+
+.desc-inside__titleNumb {
+  font-size: 35px;
+  margin: 0px;
+  margin-top: -10px;
+}
+
+.desc-inside__titleText {
+  font-size: 20px;
+  margin: 0px;
+  padding-left: 10px;
+  align-self: center;
+  margin-top: -25px;
+}
+
+.display-inside__text {
+  align-self: flex-end;
+  font-size: 12px;
+  font-family: "MontserratRegular";
+  margin: 0px;
+  margin-left: -131px;
+  padding-bottom: 10px;
+}
+
+.description-inside__update {
+  padding-top: 35px;
+  display: flex;
+  flex-direction: row;
+}
+
+.update-inside__icon {
+  display: flex;
+  flex-direction: column-reverse;
+  padding-bottom: 29px;
+}
+
+.update-inside__iconFirst {
+  width: 6.28px;
+  height: 6.42px;
+  padding-left: 16px;
+}
+
+.update-inside__iconSecond {
+  width: 18.33px;
+  height: 7.7px;
+  padding-left: 10px;
+  margin-top: -1px;
+}
+
+.update-inside__iconThird {
+  width: 31.07px;
+  height: 10.4px;
+  padding-left: 4px;
+  margin-top: -4px;
+}
+
+.update-inside__iconFourth {
+  width: 44px;
+  height: 13.12px;
+  animation-duration: 1s;
+  animation-name: blickOne;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  transform-origin: 100% 0%;
+  margin-left: -2px;
+}
+
+@keyframes blickOne {
+  0% {
+    opacity: 0;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5558864)